api4/group.go

```diff
@@ -187,26 +187,16 @@ func linkGroupSyncable(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	if groupSyncable == nil {
-		groupSyncable = &model.GroupSyncable{
-			GroupId:    c.Params.GroupId,
-			SyncableId: syncableID,
-			Type:       syncableType,
-		}
-		groupSyncable.Patch(patch)
-		groupSyncable, appErr = c.App.CreateGroupSyncable(groupSyncable)
-		if appErr != nil {
-			c.Err = appErr
-			return
-		}
-	} else {
-		groupSyncable.DeleteAt = 0
-		groupSyncable.Patch(patch)
-		groupSyncable, appErr = c.App.UpdateGroupSyncable(groupSyncable)
-		if appErr != nil {
-			c.Err = appErr
-			return
-		}
+	groupSyncable = &model.GroupSyncable{
```

Comment:
I notice line 184 calls "GetGroupSyncable" and sets the same variable. Is the call on line 184 still required, since it always being created on 190?

Reply:
@sbishel You're absolutely right that retrieval is no longer required and I hadn't noticed. Good catch, thanks.